channeld/channeld.c

```diff
@@ -1966,6 +1967,29 @@ static void send_revocation(struct peer *peer,
 
 	peer->splice_state->await_commitment_succcess = false;
 
+	/* Now that the master has persisted the new commitment advance the HSMD
+	 * and fetch the revocation secret for the old one. */
+	struct secret old_secret2;
+	struct pubkey next_point2;
```

Comment:
this is not important for functionality, but this would be good to declare all of the top functions with all the parameters declaration